src/gpuarray_blas_cuda_cublas.c

```diff
@@ -75,6 +75,7 @@ static const char *code_sgemvBH_T_a1_b1_small =                         \
   "  atomicAdd(&y[p][i*incy], yi);"                                     \
   "}\n";
 
+#if CUDA_VERSION < 8000
```

Comment:
Crap this means we need to detect the type of card and include/not include the atomicAdd for doubles